e2e/accessibility/homescreen.accessbility.spec.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jest/expect-expect */
+const {
+  launchLoop, prepareLoop, setLanguage, skipTidepoolOnboarding,
+} = require('../../utilities/prepareTest');
+const { HomeScreen } = require('../../screens/exportAllScreens');
+
+describe('Home Screen Accessibility', () => {
+  let setup;
+  beforeAll(async () => {
+    setup = {
+      language: 'enUS',
+      mockTherapySettings: true,
+    };
+  });
+  it('prepares loop', async () => {
+    await launchLoop();
+    await skipTidepoolOnboarding(setup);
+    await prepareLoop(setup);
+  });
+  describe('has HUD elements', () => {
+    let homeScreen;
+    beforeAll(async () => {
+      homeScreen = new HomeScreen(setLanguage(setup));
+    });
+    it('has a CGM pill', async () => {
+      await expect(homeScreen.CGMPill).toBeVisible();
+    });
+    it('has a loop icon', async () => {
+      // TODO
```

Comment:
hm.... I thought this already existed?

Reply:
yes, it's the smoke test converted into this. the only difference is there are a couple more checks on elements in the home screen (i.e. we weren't checking for any of the bottom button bar elements, 1 of the charts, or the cgm/pump pill)